src/html/parse-frontmatter.js

```diff
@@ -301,7 +302,11 @@ const parseFrontmatter = ({ content = {} }) => {
         join('\n'),
       );
 
-      throw new FrontmatterParsingError(`${warning}\n${sourceref}`);
+      const err = new FrontmatterParsingError(`${warning}\n${sourceref}`);
```

Comment:
Why not just always warn?

Reply:
Because you don't want to fill up the error logs in production with things that are not relevant at that time.